src/styles/global.css

```diff
@@ -62,3 +115,11 @@ html {
 .custom-list ul {
   padding-left: 10px;
 }
+@layer base {
+  * {
+    @apply border-border;
+  }
+  body {
+    @apply bg-background text-foreground;
```

Comment:
most nézem, ez is csúfította. legközelebb egyszerűbb, ha külön app a ctf-es weboldal és csak vercel.json-ban egy redirectet csinálsz rá a /arboc-on.